odoo/addons/base/models/assetsbundle.py

```diff
@@ -228,7 +228,8 @@ def clean_attachments(self, type):
         url = self.get_asset_url(
             extra='%s' % ('rtl/' if type == 'css' and self.user_direction == 'rtl' else ''),
             name=self.name,
-            type=type
+            sep='',
+            type='.%s' % type
```

Comment:
@aab-odoo Just so you know... `t-call-assets="web.assets_frontend"` deleted the attachments for assets "web.assets_frontend_minimal_js"... because the name begins by "web.assets_frontend" :sob:

Reply:
haha